bindings/node.js/kzg.cxx

```diff
@@ -126,19 +112,11 @@ Napi::Value FreeTrustedSetup(const Napi::CallbackInfo& info) {
 
 // blobToKzgCommitment: (blob: Blob, setupHandle: SetupHandle) => KZGCommitment;
 Napi::Value BlobToKzgCommitment(const Napi::CallbackInfo& info) {
-  auto env = info.Env();
-
-  size_t argument_count = info.Length();
-  size_t expected_argument_count = 2;
-  if (argument_count != expected_argument_count) {
-    return throw_invalid_arguments_count(expected_argument_count, argument_count, env);
-  }
-
-  Blob *blob = (Blob *)extract_byte_array_from_param(info, 0, "blob");
-  if (env.IsExceptionPending()) {
-    return env.Null();
+  Napi::Env env = info.Env();
+  Blob *blob = reinterpret_cast<Blob *>(get_bytes(env, info[0], BYTES_PER_BLOB, "blob"));
```

Comment:
Maybe instead of this reinterpret_cast thing around the codebase, we can introduce four wrapper functions to `get_bytes` that do: `get_blob()`, `get_commitment()`, `get_proof()`, `get_field_element()` and only use those. Then `get_bytes` becomes an internal helper function to them.

Reply:
great idea! updated